v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/SourceDbToSpannerOptions.java

```diff
@@ -71,7 +71,6 @@ public interface SourceDbToSpannerOptions extends CommonTemplateOptions {
   /* TODO: (support Sharding, PG namespaces) */
   @TemplateParameter.Text(
       order = 5,
-      regexes = {"(^[a-zA-Z0-9]+$)"},
```

Comment:
why is this removed ? In what cases will it fail ?

Reply:
This regex does not allow special characters in source db name, which is allowed in mysql. For ex, migrating a mysql db test_db will fail because _ is not allowed in the regex.